### PR TITLE
Move DBA to src/ part 3.1: Rename DBA methods

### PR DESCRIPTION
**Pull request by MrPetovan**
Part 3.1 of  #5410

This PR simply renames `DBA` methods that didn't comply with the PSR-2 code standards.

Next, `DBM` will be on the chopping block, as its existence isn't warranted anymore.

Addon pendant: https://github.com/friendica/friendica-addons/pull/657

### CODE REVIEW

**Comment by MrPetovan** [comment]
Can anyone merge this? The failed test is inconsequential.

File: src/Database/DBA.php [hunk 1]
```diff
@@ -82,29 +82,29 @@ public static function connect($serveraddr, $user, $pass, $db, $charset = null)
...
+				self::$connection->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
 				self::$connected = true;
 			} catch (PDOException $e) {
 			}
 		}
 
 		if (!self::$connected && class_exists('mysqli')) {
 			self::$driver = 'mysqli';
-			self::$db = @new mysqli($server, $user, $pass, $db, $port);
+			self::$connection = @new mysqli($server, $user, $pass, $db, $port);
```
Comment (Quix0r) [inline src/Database/DBA.php]: This change will cause conflict with my fix for missing `$port` variable, I guess?

Comment (MrPetovan) [reply src/Database/DBA.php]: Definitely.

Comment (Quix0r) [reply src/Database/DBA.php]: Then what should be merged first? Mine or this?

Comment (MrPetovan) [reply src/Database/DBA.php]: Either, it just means the other will have to be updated afterwards.

**Comment by annando** [comment]
I'm currently away from home. I will be at home in the evening. Then I will have a closer look at the PRs.

**Comment by Quix0r** [comment]
One conflict have been found.

**Pull request merged**